verdigado_attendance/models/hr_attendance.py

```diff
@@ -9,3 +9,10 @@
 
 class HrAttendance(models.Model):
     _inherit = "hr.attendance"
+
+    def _update_overtime(self, employee_attendance_dates=None):
+        """Set the flag in context to exclude public holidays"""
+        # TODO move this to glue module hr_holidays_public_attendance
+        return super(
+            HrAttendance, self.with_context(exclude_public_holidays=True)
```

Comment:
@albig that's the difference between public holidays not working and working. If you agree I'll wrap this override in the standard oca boilerplate and publish it as your module?

Reply:
https://github.com/OCA/hr-holidays/pull/87